.circleci/config.yml

```diff
@@ -191,7 +191,12 @@ jobs:
       - run: *bootstrap_old_test_suite
       - run:
           name: Run spider
-          command: cd src/api; bundle exec rake test:spider
+          command: |
+            # TODO: Remove this when switch_to_webui2 is removed
+            export BOOTSTRAP=1
```

Comment:
Can we start doing this inside the specs?

Reply:
nitpick: spider test is a minitest :)